src/jit/lsra.h

```diff
@@ -2043,6 +2071,24 @@ class RefPosition
         return delayRegFree ? nodeLocation + 1 : nodeLocation;
     }
 
+    RefPosition* getRangeEndRef()
+    {
+        if (lastUse || nextRefPosition == nullptr)
+        {
+            return this;
+        }
+        // if (nextRefPosition->lastUse)
```

Comment:
Should this `if` should be removed or uncommented... ?

Reply:
I removed the if (and the dead code below it) and added a comment.
Once this is merged, I'll file a new issue to further tune this - specifically in this area and also wrt the spills in `Span.IndexerBench`.